configs/forcedPhotCcd.py

```diff
@@ -8,3 +8,8 @@
 hscConfig = os.path.join(getPackageDir("obs_subaru"), "config", "hsc", "forcedPhotCcd.py")
 if os.path.exists(hscConfig):
     config.load(hscConfig)
+
+# Disable external calibrations: Gen3 isn't ready for these yet
+config.recalibrate.doApplyExternalPhotoCalib = False
+config.recalibrate.doApplyExternalSkyWcs = False
+config.recalibrate.doApplySkyCorr = False
```

Comment:
I don't think you need to change things here as well as in the yaml file.  In fact, I think all of these configs are not being used anymore, and should have been removed when the yaml file was added.  @natelust, can you confirm?

Reply:
Discussed with Nate off-line: you can just blow away all of the configs here except for the skymap one (or ignore them and we'll clean up on another ticket, if you prefer).